test/utils.test.js

```diff
@@ -0,0 +1,77 @@
+// import React from 'react'
+// import { format } from 'date-fns'
+
+import { isSelectable, mergeModifiers } from '../src/utils'
+import { isToday } from 'date-fns'
+
+describe('isSelectable', () => {
+  it('should return `true` for a date that is within the range of `minimumDate` and `maximumDate`', () => {
+    const testDate = new Date(2025, 10, 4)
+    const minimumDate = new Date(2025, 10, 3)
+    const maximumDate = new Date(2025, 10, 5)
+    expect(isSelectable(testDate, { minimumDate, maximumDate })).toBe(true)
+  })
+  it('should return `true` for a date that is equal to `minimumDate` or `maximumDate`', () => {
+    const minimumDate = new Date(2025, 10, 3)
+    const maximumDate = new Date(2025, 10, 5)
+    expect(isSelectable(minimumDate, { minimumDate, maximumDate })).toBe(true)
+    expect(isSelectable(maximumDate, { minimumDate, maximumDate })).toBe(true)
+  })
+  it('should return `false` for a date that is before `minimumDate` or after `maximumDate`', () => {
```

Comment:
Purely stylistic (I should add an eslint rule for this) but could you leave a blank line between the end of a block and the start of a new one? In the same way that you do within the `mergeModifiers` tests.